### ISSUE TITLE
Release profile produces bad signatures

### ISSUE THREAD

**Issue by pdolezal** [pdolezal/net.yetamine#1]
When running Maven with `release` profile, it produces invalid signatures.

The problem is Maven Repository Plugin that repackages the artifacts after signing, which breaks the signature. Unless a whole pipeline uses automated distribution management, it probably can't be solved easily. (Starting to have Maven: spending more time on dealing with these issues than with actual code development.)

**Comment by pdolezal** [comment]
I intend to drop Maven Repository Plugin completely and rather use  a simple shell script to build the bundle artifact for releasing to Maven Central.

Btw. I learned a lesson: I should not be so hasty :hushed: I managed to release artifacts that are not very useful… sorry! Alright, this time I will delay the release of the proposed version 3.0.0 (hopefully the right one) until an artifact build with the proposed version passes Maven Central checks without any problems.
